cli/admin/src/main/java/io/pravega/cli/admin/readerGroup/ParseReaderGroupStreamCommand.java

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.readerGroup;
+
+import io.pravega.cli.admin.AdminCommand;
+import io.pravega.cli.admin.AdminCommandState;
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.segmentstore.ReadSegmentRangeCommand;
+import io.pravega.cli.admin.utils.ConfigUtils;
+import io.pravega.cli.admin.utils.FileHelper;
+import io.pravega.client.admin.impl.ReaderGroupManagerImpl;
+import io.pravega.client.control.impl.Controller;
+import io.pravega.client.state.impl.UpdateOrInitSerializer;
+import io.pravega.client.stream.impl.StreamSegments;
+import io.pravega.controller.server.SegmentHelper;
+import io.pravega.controller.server.security.auth.GrpcAuthHelper;
+import io.pravega.shared.NameUtils;
+import io.pravega.shared.protocol.netty.PravegaNodeUri;
+import io.pravega.shared.protocol.netty.WireCommandType;
+import io.pravega.shared.protocol.netty.WireCommands;
+import lombok.Cleanup;
+import lombok.val;
+import org.apache.curator.framework.CuratorFramework;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileInputStream;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+
+public class ParseReaderGroupStreamCommand extends AdminCommand {
```

Comment:
Maybe this could be placed as part of the Controller commands, as we have others like "describe reader groups" which are similar, from a user perspective (I know that others use REST API, but this is not seen by the user).